src/components/header/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+
+import './header.css';
+
+function Header() {
+
+    const [userName, setUserName] = useState('');
+
+    return (
+        <section id="header">
+            <nav>
+                <div>
+                    <NavLink to="/">Home</NavLink> {' '}
```

Comment:
What are all of these `{' '}`?

Reply:
If the spaces are used to space create space between your elements rather use css to style it correctly.